server/.gitignore

```diff
@@ -2,3 +2,6 @@ bin
 
 # ignore code coverage information
 cover.out
+
+# for packaging
+server.tar.gz
```

Comment:
This can be removed if we are already ignoring the `out/` folder

Reply:
Fixed in 7d80cb4.